CONTRIBUTING.adoc

```diff
@@ -9,7 +9,7 @@
 
 The guides toolchain is heavily dependent on AsciiDoc and Node.
 
-. Install your favourite tool for editing AsciiDoc (for example, Atom).
+. Install your favourite tool for editing AsciiDoc (for example, VS Code or IntelliJ).
```

Comment:
```suggestion
. Install your favourite tool for editing AsciiDoc (for example, VS Code or IntelliJ IDEA).
```

Reply:
Yes, I think in this context `IntelliJ` is OK.  

See existing section on `Adding support for quickstart.yml to your IDE`, which uses `VS Code` and `IntelliJ` :)